ui/lib/path.js

```diff
@@ -18,6 +18,7 @@ export const registerUser = () => "/user-registration";
 export const createIdentity = _params => "/identity/new";
 
 export const orgs = id => `/orgs/${id}`;
+export const orgRegistration = id => `/orgs/${id}/register`;
```

Comment:
I don't think we know the `id` upfront before going through the registration flow, shouldn't it be `/orgs/register`?